woocommerce-gutenberg-products-block.php

```diff
@@ -286,3 +286,25 @@ function woocommerce_blocks_plugin_outdated_notice() {
 }
 
 add_action( 'admin_notices', 'woocommerce_blocks_plugin_outdated_notice' );
+
+/**
+ * Register the Interactivity API scripts. These files are enqueued when a block
+ * defines `wp-directives-runtime` as a dependency.
+ */
+function wp_directives_register_scripts() {
+	wp_register_script(
+		'wp-directives-vendors',
+		plugins_url( 'build/wp-directives-vendors.js', __FILE__ ),
+		array(),
+		'1.0.0',
+		true
+	);
+	wp_register_script(
+		'wp-directives-runtime',
+		plugins_url( 'build/wp-directives-runtime.js', __FILE__ ),
+		array( 'wp-directives-vendors' ),
+		'1.0.0',
+		true
+	);
```

Comment:
```suggestion
	wp_enqueue_script( 'wp-directives-runtime' );
```

The script `wp-directives-runtime` isn't enqueued.